.github/workflows/build-ghcr.yml

```diff
@@ -1,10 +1,7 @@
----
-name: Build GHCR
+name: Build for GitHub Container Registry
 
 on:
   push:
-    paths-ignore:
-      - ".github/workflows/**"
```

Comment:
Why stop ignoring `.github/workflows/**`?

Reply:
I see this a lot and don't understand why. This would avoid running this on pushes should there only be changes on that path, but ... who cares? We've overcomplicated it and should build on changes, period.